sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -959,7 +959,7 @@ inline pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
     InfoType = UR_DEVICE_INFO_IMAGE_SRGB;
     break;
   case PI_EXT_ONEAPI_DEVICE_INFO_COMMAND_BUFFER_SUPPORT:
-    InfoType = (ur_device_info_t)UR_EXT_DEVICE_INFO_COMMAND_BUFFER_SUPPORT;
+    InfoType = (ur_device_info_t)UR_DEVICE_INFO_EXP_COMMAND_BUFFER_SUPPORT;

```

Comment:
Noting that this needs removed to reflect https://github.com/oneapi-src/unified-runtime/pull/560/commits/412ebda36c5648609d749da03f520a7be3fed2e4 and a string returned from the device extension query as per https://github.com/oneapi-src/unified-runtime/pull/560#discussion_r1219760445

Reply:
This has been removed now.